### PR TITLE
Add example with usual system

### PR DESCRIPTION
**Pull request by torkleyy**
As there has been some confusion how to do it.

### CODE REVIEW

File: examples/basic.rs [hunk 1]
```diff
@@ -60,31 +68,27 @@ fn main() {
     // We have 5 entities and this will print only 4 of them
     println!("Only entities with CompBool present:");
     planner.run0w1r(|b: &CompBool| {
-        println!("Entity {}", b.0);
-    });
+                        println!("Entity {}", b.0);
```
Comment (kvark) [inline examples/basic.rs]: messed up indentation

Comment (torkleyy) [inline examples/basic.rs]: I'm not using tabs, hat's rustfmt...

Comment (torkleyy) [inline examples/basic.rs]: I thought rustfmt is a widely used tool, why does it mess up closures?

Comment (derekdreery) [inline examples/basic.rs]: It's indenting 4 spaces from `|b: &CompBool|`

**Pull request merged**